01_loaddata.R

```diff
@@ -1,25 +1,78 @@
 
-# methdology for Redknot conversion to R
+# Redknot Methodology Conversion to R
 
+# Load R Packages
 library(fs)
 library(dplyr)
 library(readr)
 library(sf)
-library(terra) #testing line
-library(lidR) # cam test
+library(terra)
 
-# 1) read in csv from 01_inputs folder
+## DATA PRE-PROCESSING ##
+# Read CSV Data
+birds <- read_csv(path("01_inputs", "DS 2 _P7165_Con_213835_250122134729.csv"),name_repair = "unique",locale=locale(encoding="latin1"))
 
-bd <- read_csv(path("01_inputs", "DS 2 _P7165_Con_213835_250122134729.csv"),name_repair = "unique",locale=locale(encoding="latin1"))
+# Convert to SF Object
+birds_sf <- st_as_sf(birds, coords = c("Longitude", "Latitude"), crs = 4326)
 
+# Filter Out Bad Data and Sort by Date Ascending
+birds_sf <- birds_sf %>% filter(CRC != 'Fail')
 
+birds_sf <- birds_sf %>% arrange(Date)
```

Comment:
I would suggest we convert this to a date/time column. This will help further down with the sorting below. I also added the prior lat and long to the dataset to use in the bearing calculation later on
```suggestion
birds_sf <- birds_sf |> 
    dplyr::mutate(datetime = dmy_hms(paste(Date, Time, sep = " ")))

birds_sf <- birds_sf %>% arrange(datetime)

birds_sf <- birds_sf |> 
   mutate(location.long_prior = lag(Longitude, 1L),
          location.lat_prior = lag(Latitude, 1L))

```